configs/components/oifs/oifs.yaml

```diff
@@ -100,6 +101,15 @@ compile_infos:
             branch: foci_conserv
             comp_command: cd make; ../fcm/bin/fcm make -v -j24 -f oifs.cfg
             destination: 40r1
+          43r3-master:
+            requires:
+            - oasis3mct-4.0-awicm3-frontiers
+            branch: master
+            comp_command: export OIFS_TOPLEVEL_DIR=${model_dir}; cd make; ../fcm/bin/fcm make
+              -v -j8 -f oifs.cfg ; cp -p esm/oifs/bin/master.exe esm/oifs/bin/oifs
```

Comment:
I did not understand the copying of the binary `master.exe` to `oifs`. Is there a reason for the `master.exe` to stay in the same directory? What about linking> I never user OIFS, btw.

Reply:
probably better to mv actually. Less confusion about why there might be two executable. I'll fix that.